src/core/symbology-ng/qgssymbollayerv2.cpp

```diff
@@ -225,6 +225,8 @@ QgsMarkerSymbolLayerV2::QgsMarkerSymbolLayerV2( bool locked )
 QgsLineSymbolLayerV2::QgsLineSymbolLayerV2( bool locked )
     : QgsSymbolLayerV2( QgsSymbolV2::Line, locked )
     , mWidthUnit( QgsSymbolV2::MM )
+    , mOffset( 0 )
+    , mOffsetUnit( QgsSymbolV2::MM )
 {
```

Comment:
@vmora - neither QgsMarkerLineSymbolLayerV2 nor QgsSimpleLineSymbolLayerV2 call the constructor for QgsLineSymbolLayerV2, so these properties won't be correctly initialised. I wonder if this reveals a larger bug, since it seems that none of the symbol layer types call their base class' constructor...